### DISCUSSION TITLE
gtk::TextTag

### DISCUSSION THREAD

**Discussion by 87flowers** [Relm4/Relm4#680] [Q&A] [answered]
What is a good way of constructing text tag tables, and declaring text buffers with text tags / pixmaps be in Relm4?
There's the obvious non-idiomatic way of doing it, but was wondering if there was any intended way for this to be done.

**Comment by AaronErhardt** [comment, marked as answer]
I'm not sur what you want exactly, but if you want to create a table of something either factories or `TypedGridView` might be a good solution. Otherwise there's just regular gtk-rs left. But feel free to add more details or ask in the Matrix room.
